marklogic-sesame/src/test/java/com/marklogic/semantics/sesame/MarkLogicRepositoryConnectionTest.java

```diff
@@ -1224,7 +1224,7 @@ public void testNestedConnections()
         config.setPort(port);
         config.setUser(user);
         config.setPassword(password);
-        config.setAuth("DIGEST");
+        config.setAuth("BASIC");
```

Comment:
The swtich to BASIC is probably fine for this.  We have a long-standing requirement to support "DIGEST" though, and we've found that this is something that is not universally supported in HTTP clients.  Thus we will probably have to keep this "DIGEST" and see what issues, perhaps, you ran into.